examples/oriented_images.js

```diff
@@ -0,0 +1,107 @@
+var viewerDiv = document.getElementById('viewerDiv');
+
+// Instanciate iTowns GlobeView*
+var globeView = new itowns.GlobeView(viewerDiv, positionOnGlobe, {
+    renderer: renderer,
+    handleCollision: false,
+    sseSubdivisionThreshold: 6,
+    noControls: true,
+});
+
+// globeView.controls.minDistance = 0;
+var pictureInfos;
+var coord;
+var rotationMatrix;
+// eslint-disable-next-line no-unused-vars
+var plane;
+
+function addLayerCb(layer) {
+    return globeView.addLayer(layer);
+}
+
+// Define projection that we will use (taken from https://epsg.io/3946, Proj4js section)
+itowns.proj4.defs('EPSG:3946',
+    '+proj=lcc +lat_1=45.25 +lat_2=46.75 +lat_0=46 +lon_0=3 +x_0=1700000 +y_0=5200000 +ellps=GRS80 +towgs84=0,0,0,0,0,0,0 +units=m +no_defs');
+
+// Add one imagery layer to the scene
+// This layer is defined in a json file but it could be defined as a plain js
+// object. See Layer* for more info.
+promises.push(itowns.Fetcher.json('./layers/JSONLayers/Ortho.json').then(addLayerCb));
+
+// Add two elevation layers.
+// These will deform iTowns globe geometry to represent terrain elevation.
+promises.push(itowns.Fetcher.json('./layers/JSONLayers/WORLD_DTM.json').then(addLayerCb));
+promises.push(itowns.Fetcher.json('./layers/JSONLayers/IGN_MNT_HIGHRES.json').then(addLayerCb));
+
+pictureInfos = {
+    panoramic: {
+        latitude: 45.9228208442959,
+        height: 4680.55588294683,
+        longitude: 6.83256920100156,
+        azimuth: 526.3900321920207,
+        roll: 2.1876227239518276,
+        tilt: -11.668910605126001,
+        // azimuth_: 0,
+        // roll_: 0,
+        // tilt_: 0,
+        image: './orientedImages/MontBlanc.jpg',
+    },
+    camera: {
+        size: [6490, 4408],
+        focale: 1879.564256099287 * 4.4,
+        ppaX: 3245,
+        ppaY: 2204,
+        // Define the referential : Aircarft orientation
+        // Starting from from ENH orientation ( X to east, Y to the north, Z is the vertical)
+        // We are creating the classic aircraft axises, using lookAt and Up.
+        // First we look at the north, (north is on the Y axis in ENH orientation),
+        // so Z axis will point to the north
+        lookAt: new itowns.THREE.Vector3(0, 1, 0),
+        // then set the up vector to look to the ground,
+        // wich is the opposition of Z in ENH orientation, so Y axis will point to the ground.
+        up: new itowns.THREE.Vector3(0, 0, -1),
+        // after that, X axis will naturally point the east.
+    },
+    distance: 4000,
+    opacity: 0.8,
+};
+
+function parseAircraftConventionOrientationToMatrix(panoramic) {
+    var euler = new itowns.THREE.Euler(
+        itowns.THREE.Math.degToRad(panoramic.tilt),
+        itowns.THREE.Math.degToRad(panoramic.azimuth),
+        itowns.THREE.Math.degToRad(panoramic.roll),
+        'ZYX');
+
+    return new itowns.THREE.Matrix4().makeRotationFromEuler(euler);
+}
+
+function toCoordGeographic(panoramic) {
```

Comment:
This one line function is only used once: let's get rid of it and use its code directly at the calling site.